pkg/minikube/shell/shell.go

```diff
@@ -129,20 +129,25 @@ REM @FOR /f "tokens=*" %%i IN ('%s') DO @%%i
 	},
 }
 
-var defaultSh = "bash"
-var defaultShell shellData = shellConfigMap[defaultSh]
+// DefaultShellName is default shell which will be used in case of shell detection error
+const DefaultShellName = "bash"
```

Comment:
change the name, "shell.DefaultShellName" stutters.

how about shell.Default ?